src/components/Board.tsx

```
@@ -4,17 +4,10 @@ import { useRef, useEffect } from 'react';
-// 보드 상태에 대한 타입 정의
-interface BoardStone {
-  stone: StoneEnum;
-  xPos: number;
-  yPos: number;
-}
-
 interface BoardProps {
   size: number;
-  boardState: BoardStone[][];
-  lastMove: {x: number, y: number} | null;
+  boardState: { xPos: number; yPos: number; stone: number }[][];
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)